modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -176,11 +197,15 @@ struct IEUnit {
 
     explicit IEUnit(const cv::gapi::ie::detail::ParamDesc &pp)
         : params(pp) {
-
+#if INF_ENGINE_RELEASE < 2020000000  // < 2020.1
         IE::CNNNetReader reader;
         reader.ReadNetwork(params.model_path);
         reader.ReadWeights(params.weights_path);
         net = reader.getNetwork();
+#else // >= 2020.1
+        InferenceEngine::Core& core = getCore(params.device_id);
```

Comment:
I doubt if we need a distinct `ie::Core` object per every `device` we use. cc: @ilya-lavrenov